<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1400</width>
    <height>900</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <layout class="QVBoxLayout" name="mainLayout">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="sizeConstraint">
       <enum>QLayout::SetNoConstraint</enum>
      </property>
      <item>
       <layout class="QHBoxLayout" name="optionsLayout">
        <item>
         <layout class="QHBoxLayout" name="slicOptionsLayout">
          <item>
           <layout class="QVBoxLayout" name="slicOptionsNames">
            <item>
             <widget class="QLabel" name="slicIterationsOptionName">
              <property name="text">
               <string>Iterations</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="slicRulerOptionName">
              <property name="text">
               <string>Ruler</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="slicSizeOptionName">
              <property name="text">
               <string>Size</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="slicEnforceConnectivityOtionName">
              <property name="text">
               <string>Enforce connectivity</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QVBoxLayout" name="slicOptionValues">
            <item>
             <widget class="QSlider" name="slicIterationsSlider">
              <property name="maximumSize">
               <size>
                <width>1000</width>
                <height>16777215</height>
               </size>
              </property>
              <property name="minimum">
               <number>5</number>
              </property>
              <property name="maximum">
               <number>100</number>
              </property>
              <property name="value">
               <number>10</number>
              </property>
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="invertedAppearance">
               <bool>false</bool>
              </property>
              <property name="invertedControls">
               <bool>false</bool>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QSlider" name="slicRulerSlider">
              <property name="maximumSize">
               <size>
                <width>1000</width>
                <height>16777215</height>
               </size>
              </property>
              <property name="minimum">
               <number>5000</number>
              </property>
              <property name="maximum">
               <number>20000</number>
              </property>
              <property name="singleStep">
               <number>10</number>
              </property>
              <property name="value">
               <number>10000</number>
              </property>
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="tickPosition">
               <enum>QSlider::NoTicks</enum>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QSlider" name="slicSizeSlider">
              <property name="maximumSize">
               <size>
                <width>1000</width>
                <height>16777215</height>
               </size>
              </property>
              <property name="minimum">
               <number>8</number>
              </property>
              <property name="maximum">
               <number>36</number>
              </property>
              <property name="value">
               <number>16</number>
              </property>
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QCheckBox" name="slicEnforceConnectivityCheckBox">
              <property name="text">
               <string/>
              </property>
              <property name="checked">
               <bool>true</bool>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="merwOptionsSlider">
          <item>
           <layout class="QVBoxLayout" name="merwOptionNames">
            <item>
             <widget class="QLabel" name="merwLocalizeGraphOptionName">
              <property name="text">
               <string>LocalizeGraph</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="merwColorDistanceOptionName">
              <property name="text">
               <string>Color/Distance value</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="merwEpsilonOptionName">
              <property name="text">
               <string>Epsilon</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="label">
              <property name="text">
               <string/>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QVBoxLayout" name="merwOptionsValues">
            <item>
             <widget class="QCheckBox" name="merwLocalizeGraphCheckbox">
              <property name="text">
               <string/>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QSlider" name="merwColorDistanceSlider">
              <property name="maximumSize">
               <size>
                <width>1000</width>
                <height>16777215</height>
               </size>
              </property>
              <property name="maximum">
               <number>100</number>
              </property>
              <property name="value">
               <number>50</number>
              </property>
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QSlider" name="merwEpsilonSlider">
              <property name="minimum">
               <number>0</number>
              </property>
              <property name="maximum">
               <number>30</number>
              </property>
              <property name="value">
               <number>5</number>
              </property>
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="label_2">
              <property name="text">
               <string/>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QVBoxLayout" name="statisticsLayout">
          <item>
           <layout class="QHBoxLayout" name="numberOfNodesLayout">
            <item>
             <widget class="QLabel" name="numberOfNodesTextLabel">
              <property name="text">
               <string>Number of nodes: </string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="numberOfNodesLabel">
              <property name="text">
               <string>XXX</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <widget class="QLabel" name="label_4">
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
          <item>
           <layout class="QHBoxLayout" name="calculateLayout">
            <item>
             <widget class="QPushButton" name="calculateButton">
              <property name="enabled">
               <bool>false</bool>
              </property>
              <property name="text">
               <string>Calculate</string>
              </property>
              <property name="checkable">
               <bool>false</bool>
              </property>
              <property name="checked">
               <bool>false</bool>
              </property>
              <property name="flat">
               <bool>false</bool>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="imagesLayout">
        <item>
         <layout class="QVBoxLayout" name="imagesCol0Layout">
          <item>
           <widget class="QLabel" name="image0View">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text">
             <string>TextLabel</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="image1View">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text">
             <string>TextLabel</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QVBoxLayout" name="imagesCol1Layout">
          <item>
           <widget class="QLabel" name="image2View">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text">
             <string>TextLabel</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="image3View">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text">
             <string>TextLabel</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1400</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionLoad"/>
   </widget>
   <addaction name="menuFile"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionLoad">
   <property name="text">
    <string>Load</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
